src/Admin/AppAdmin.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Admin;
+
+use Sulu\Bundle\AdminBundle\Admin\Admin;
+use Sulu\Bundle\AdminBundle\Admin\View\FormViewBuilderInterface;
+use Sulu\Bundle\AdminBundle\Admin\View\ToolbarAction;
+use Sulu\Bundle\AdminBundle\Admin\View\ViewCollection;
+use Sulu\Bundle\ContactBundle\Admin\ContactAdmin;
+
+class AppAdmin extends Admin
+{
+    public function configureViews(ViewCollection $viewCollection): void
+    {
+        if ($viewCollection->has('sulu_contact.contact_add_form.details')) {
```

Comment:
This will add the toolbar action to the form for creating new contacts only. It will **not affect** the form for editing existing contacts.